interface/wx/versioninfo.h

```diff
@@ -33,6 +33,52 @@ enum class wxVersionContext
     provide version information about your own libraries, or other libraries
     that you use.
 
+    The following example shows how to get most of wxWidgets's library information
+    and format it all into a string.
+    (Note that this assumes all libraries are being linked and @c wxInitAllImageHandlers()
+     has already been called.)
+
+    @code
+    // Consistently format library info with the name and version number.
+    const auto formatLibInfo = [](const auto& libInfo)
+    {
+        return wxString::Format(_("%s - version %s"), libInfo.GetName(),
+                                libInfo.GetNumericVersionString());
+    };
+
+    // Get wxWidgets's version--as well as its submodules--and sort them.
+    std::vector<wxString> allLibInfo;
+    allLibInfo.push_back(formatLibInfo(wxGetLibraryVersionInfo()));
+    allLibInfo.push_back(formatLibInfo(wxTIFFHandler::GetLibraryVersionInfo()));
+    allLibInfo.push_back(formatLibInfo(wxJPEGHandler::GetLibraryVersionInfo()));
+    allLibInfo.push_back(formatLibInfo(wxPNGHandler::GetLibraryVersionInfo()));
+    allLibInfo.push_back(formatLibInfo(wxWEBPHandler::GetLibraryVersionInfo()));
+    allLibInfo.push_back(formatLibInfo(wxRegEx::GetLibraryVersionInfo()));
+    allLibInfo.push_back(formatLibInfo(wxXmlDocument::GetLibraryVersionInfo()));
+    allLibInfo.push_back(formatLibInfo(wxStyledTextCtrl::GetLibraryVersionInfo()));
+    allLibInfo.push_back(formatLibInfo(wxWebSession::GetDefault().GetLibraryVersionInfo()));
+    allLibInfo.push_back(formatLibInfo(wxGetZlibVersionInfo()));
+    allLibInfo.push_back(formatLibInfo(wxWebView::GetBackendVersionInfo()));
```

Comment:
```suggestion
    std::vector<wxString> allLibInfo {
        formatLibInfo(wxGetLibraryVersionInfo()),
        formatLibInfo(wxTIFFHandler::GetLibraryVersionInfo()),
        formatLibInfo(wxJPEGHandler::GetLibraryVersionInfo()),
        formatLibInfo(wxPNGHandler::GetLibraryVersionInfo()),
        formatLibInfo(wxWEBPHandler::GetLibraryVersionInfo()),
        formatLibInfo(wxRegEx::GetLibraryVersionInfo()),
        formatLibInfo(wxXmlDocument::GetLibraryVersionInfo()),
        formatLibInfo(wxStyledTextCtrl::GetLibraryVersionInfo()),
        formatLibInfo(wxWebSession::GetDefault().GetLibraryVersionInfo()),
        formatLibInfo(wxGetZlibVersionInfo()),
        formatLibInfo(wxWebView::GetBackendVersionInfo())};
```
unless C++03 support is really required ;-)